src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,60 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+type Props = {
+  setStatus: (status: string) => void;
+  setQuery: (status: string) => void;
```

Comment:
The `setQuery` function in the `Props` type should accept a `query` parameter instead of a `status`. Update the parameter name to reflect its purpose more accurately, e.g., `(query: string) => void;`.